src/components/EachTodo.js

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import { setTodo } from "../hooks/setTodo";
+
+function EachTodo({ index, todo, todoLists, setTodoLists }) {
+  const [checkState, setCheckState] = useState(todo.checked);
+  const checkboxChanged = (e) => {
+    const updatedTodoLists = [...todoLists];
+    updatedTodoLists[index] = {
+      ...updatedTodoLists[index],
+      checked: !updatedTodoLists[index].checked,
+    };
+    setTodoLists(updatedTodoLists);
+    setCheckState((prev) => !prev);
+  };
+  const deleteTodo = () => {
+    const updatedTodoLists = [...todoLists];
+    updatedTodoLists.splice(index, 1);
```

Comment:
이부분은 filter함수를 이용해서
```suggestion
    const updatedTodoLists = todoLists.filter((_, idx) => idx !== index);
```
처럼 사용할 수 있을 것 같아요

Reply:
좋은 의견 감사드립니다~!!